src/components/Icon/France.tsx

```diff
@@ -0,0 +1,33 @@
+export const France = ({ width = 32, height = 24 }): JSX.Element => (
```

Comment:
Maybe it will be better to also add SVGProps<SVGElement> TS props definition as in other svgs?